OpalImagePicker/Source/UIKit Extensions.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  UIKit Extensions.swift
```

Comment:
I would prefer to have files not have spaces. You could rename it UIKit+Extensions.swift or two files UIView+Helpers.swift, Bundle+Helpers.swift